xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecCCText.cpp

```diff
@@ -35,27 +35,14 @@ CDVDOverlayCodecCCText::CDVDOverlayCodecCCText() : CDVDOverlayCodec("CC Text Sub
   m_changePrevStopTime = false;
 }
 
-CDVDOverlayCodecCCText::~CDVDOverlayCodecCCText()
-{
-  Dispose();
-}
+CDVDOverlayCodecCCText::~CDVDOverlayCodecCCText() = default;
```

Comment:
can you delete all these decostructors since are not used (on each overlay codec)?

Reply:
I don't have a full view of the code atm, but sometimes it's necessary to have these empty destructors in cpp files to fix link errors. They wouldn't be out of place if they stayed.

Reply:
It doesn't seem to cause linker errors in this case so I removed them. Also thanks for testing @CastagnaIT!